### ISSUE TITLE
Error on "make install" MAC OSX 10.12.3

### ISSUE THREAD

**Issue by Alutoe** [MIT-LCP/physionet#36]
Getting the following error when running command `make install` I'm running MAC OSX 10.12.3.

`cd lib;	     /Applications/Xcode.app/Contents/Developer/usr/bin/make clean install
rm -f wfdbinit.o annot.o signal.o calib.o wfdbio.o libwfdb.* *.dll *~
/Applications/Xcode.app/Contents/Developer/usr/bin/make clean	    # force recompilation since config may have changed
rm -f wfdbinit.o annot.o signal.o calib.o wfdbio.o libwfdb.* *.dll *~
/Applications/Xcode.app/Contents/Developer/usr/bin/make all
sed "s+DBDIR+/usr/local/database+" <wfdblib.h0 >wfdblib.h
gcc -arch i386 -arch x86_64 -fPIC -fno-common -g -O -DWFDB_MAJOR=10 -DWFDB_MINOR=5 -DWFDB_RELEASE=24 -DNOVALUES_H `curl-config --cflags` -I/usr/local/include   -c -o wfdbinit.o wfdbinit.c
In file included from wfdbinit.c:33:
In file included from ./wfdblib.h:234:
In file included from /Users/Alyssa/anaconda/include/curl/curl.h:35:
/Users/Alyssa/anaconda/include/curl/curlrules.h:143:6: error: '__curl_rule_01__' declared as an array
      with a negative size
    [CurlchkszEQ(long, CURL_SIZEOF_LONG)];
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/Alyssa/anaconda/include/curl/curlrules.h:132:27: note: expanded from macro 'CurlchkszEQ'
#define CurlchkszEQ(t, s) sizeof(t) == s ? 1 : -1
                          ^~~~~~~~~~~~~~~~~~~~~~~
/Users/Alyssa/anaconda/include/curl/curlrules.h:153:6: error: '__curl_rule_02__' declared as an array
      with a negative size
    [CurlchkszEQ(curl_off_t, CURL_SIZEOF_CURL_OFF_T)];
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/Alyssa/anaconda/include/curl/curlrules.h:132:27: note: expanded from macro 'CurlchkszEQ'
#define CurlchkszEQ(t, s) sizeof(t) == s ? 1 : -1
                          ^~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.
make[2]: *** [wfdbinit.o] Error 1
make[1]: *** [install] Error 2
make: *** [install] Error 2`

**Comment by zezeko** [comment]
@Alutoe I encountered the same problem. have you found a workaround since then? i am using mac os 10.13.4.

**Comment by Alutoe** [comment]
No, I have not found a workaround. I ended up giving up on the problem. I hope you have better luck.

**Comment by tompollard** [comment]
What dataset are you trying to load? Where did you find the makefile that you are attempting to run?

**Comment by zezeko** [comment]
I was trying to install 10.6 but the original post is from one year ago. Thanks for the response. I have installed the package successfully.

**Comment by bemoody** [comment]
That's good to hear.  I'll try to fix this in the next version of WFDB.
